apps/chat/src/components/Chat/ChangePathDialog.tsx

```diff
@@ -198,12 +198,13 @@ export const ChangePathDialog = ({
             newAddedFolderId: newFolderId,
             featureType: FeatureType.File,
           }}
-          handleToggleFolder={handleToggleFolder}
+          handleFolderSelect={handleFolderSelect}
```

Comment:
we still need to handle `toggle`, not only `select/open`. Especially cases `setIsAllFoldersOpened`

Reply:
Actually toggle handled inside select folder
Moreover you cannot toggle without selecting